app/views/track.js

```diff
@@ -118,6 +127,42 @@ const TrackView = Mn.LayoutView.extend({
         e.stopPropagation();
     },
 
+    setClass() {
+        this.$el.removeClass();
+        this.order = [];
+
+        // Check if track is in upNext
+        let tracks = application.upNext.get('tracks');
+        tracks.each((track, index) => {
+            if (track.get('_id') == this.model.get('_id')) {
+                this.$el.addClass(' playlist-upNext');
```

Comment:
`$.addClass` auto add spaces, no need for an extra prefix space on ` playlist-upNext`